airflow/models.py

```diff
@@ -1961,6 +1965,9 @@ class derived from this one results in the creation of a task object,
     :type resources: dict
     :param run_as_user: unix username to impersonate while running the task
     :type run_as_user: str
+    :type notifiers: dict
+    :param notifiers: A dict of AirflowNotifiers which are called whenever the
```

Comment:
Why a dict? Seems to me like a list of instantiated Notifier is what you need here. What dictates what the keys of this dict should be?